modules/about-kserve-deployment-modes.adoc

```diff
@@ -28,26 +25,24 @@ Advantages:
 
 Disadvantages:
 
-* Has customization limitations: 
-** Serverless is limited to Knative, such as when mounting multiple volumes.
-
-* Dependency on Knative for scaling: 
+* Has customization limitations:
+** Serverless is backed by Knative and implicitly inherits the same design choices, such as when mounting multiple volumes.
+* Dependency on Knative for scaling:
 ** Introduces additional complexity in setup and management compared to traditional scaling methods.
+* Cluster scoped component:
+** If the cluster already has Serverless configured, you must manually configure the cluster to make it work with {productname-short}.
 
-== Raw deployment mode
+== Standard mode
 
 Advantages:
 
-* Enables deployment with Kubernetes resources, such as `Deployment`, `Service`, `Ingress`, and `Horizontal Pod Autoscaler`: 
-** Provides full control over Kubernetes resources, allowing for detailed customization and configuration of deployment settings.
+* Enables deployment with Kubernetes resources, such as `Deployment`, `Service`, `Ingress`, and `Horizontal Pod Autoscaler`, without additional dependencies like {org-name} Serverless and {org-name} Service Mesh.
 
-* Unlocks Knative limitations, such as being unable to mount multiple volumes: 
+* Enables all traditional Deployment/Pod configuration not available via Knative, such as being unable to mount multiple volumes:
```

Comment:
```suggestion
* Enables traditional Deployment/Pod configuration not available via Knative, such as being unable to mount multiple volumes:
```

Just because I'm not sure if it is really "all".